### DISCUSSION TITLE
Getting 1994 vibes, slow download speed then crash.

### DISCUSSION THREAD

**Discussion by kevinwalter** [LegacyUpdate/LegacyUpdate#70] [Questions & Help] [answered]
Hi,

When trying to install Legacy Update on my XP SP3 VM it does not download it.
It tries to, but eventually crashes.

Any ideas?

![image](https://user-images.githubusercontent.com/979768/209435092-4502fa3d-7bb2-46c9-9c3d-ac3aaf01643f.png)

**Comment by kevinwalter** [comment, marked as answer]
Update: It had something to do with Ubiquiti Thread Management. Disabled it and it works again.


![image](https://user-images.githubusercontent.com/979768/209446944-ff44607d-b328-47f6-bdbf-b01c484cce41.png)

**Reply by kirb** [reply]
Could be because it’s unencrypted http traffic, so it’s scanning every packet, whereas there’s not quite as much it can detect in TLS-encrypted traffic. In my experience the earlier USGs struggle to do threat management/deep packet inspection, the CPU is very weak and the cooling solution isn’t adequate. I believe the UDMs are far better at it though.

**Comment by kozality** [comment]
I ran into this as well. Rather than disable Threat Management, I went into my logs and found the entries for legacyupdate.net, and allowed those particular threat signatures. The service seems to be working fine now.
